src/Orckestra.Composer.Website/App_Data/Razor/Composer/BrowsingCategories/Summary.cshtml

```diff
@@ -26,11 +26,14 @@
 @{
     var summary = Data.Get<ICategoryPageSummaryMeta>().FirstOrDefault(i => i.PageId == CurrentPageNode.Id);
     var model = BrowseCategoryRequestContext.GetViewModelAsync().GetAwaiter().GetResult();
+
+    var tabs = new[] { new { Total = model.ProductSearchResults.TotalCount }};
+
     if (summary == null)
     {
-        <div class="@(FullWidth ? "container-fluid": "")  page-summary" 
+        <div class="@(FullWidth ? "container-fluid": "")  page-summary"
              data-oc-controller="Product.SearchSummary"
-             id="vueSearchSummary" data-context="@JsonConvert.SerializeObject(model.ProductSearchResults)"
+             id="vueSearchSummary" data-context="@JsonConvert.SerializeObject(tabs)"
```

Comment:
can we use separate Browsing.SearchSummary to avoid this confusing with Tabs which are absent here